pdr_backend/pdr_dashboard/util/data.py

```diff
@@ -126,3 +131,95 @@ def find_with_key_value(
             return obj
 
     return None
+
+
+def get_feeds_stat_with_contract(
+    contract: str, feed_stats: List[Dict[str, Any]]
+) -> Dict[str, Union[float, int]]:
+    result = find_with_key_value(feed_stats, "contract", contract)
+
+    if result:
+        return {
+            "avg_accuracy": float(result["avg_accuracy"]),
+            "avg_stake_per_epoch_(OCEAN)": float(result["avg_stake"]),
+            "volume_(OCEAN)": float(result["volume"]),
+        }
+
+    return {
+        "avg_accuracy": 0,
```

Comment:
I'm thinking maybe we should have some defaults in the result itself? Just an idea, not part of the main code review